src/dpq2/conv/time.d

```diff
@@ -68,28 +86,108 @@ if( is( T == TimeStampWithoutTZ ) )
     );
 }
 
+/++
+    Structure to represent PostgreSQL's Timestamp without time zone type.
+    It is assumed to be in Local Time.
```

Comment:
> It is assumed to be in Local Time.

This is wrong assumption.

Reply:
Assumed from this line in PG docs:

> Conversions between timestamp without time zone and timestamp with time zone normally assume that the timestamp without time zone value should be taken or given as timezone local time. A different time zone can be specified for the conversion using AT TIME ZONE.

But I can remove the line for sure or replace it with the form from PG docs so it is the same

Reply:
> Assumed from this line in PG docs

No. Careful PG doc assumes this only for conversions between timestamp without time zone and timestamp with time zone, but not for representation.